Cargo.toml

```diff
@@ -44,6 +44,7 @@ ed25519-dalek = "1.0.1"
 jsonrpsee-http-client = "0.15.1"
 jsonrpsee-core = "0.15.1"
 regex = "1.6.0"
+wasm-opt = "0.110.1"
```

Comment:
@willemneal this appears to be very similar to importing binaryen which you added in a similar way in #226, but rust-analyzer appears to still work with wasm-opt where-as I know you mentioned in #226 binaryen caused it to break and you had some concerns with it being unmaintained. Wdyt of using wasm-opt instead?

Reply:
I much prefer it! Binaryen was unmaintained and clearly a lot of work including using cxx went into making wasm-opt!

Reply:
@willemneal Fyi I'm still interested in #226, which goes a step further as having an entire build process built in. Independent of that I think we'll still want a command like this, to optimize a wasm file. So I think #236 and #226 should co-exist.

Reply:
Great! I'm about to finish it up.  I'll strip out the binaryen stuff and I'll rebase to this one to reuse wasm-opt.